### ISSUE TITLE
Can't export model in opencv..?

### ISSUE THREAD

**Issue by ganesharavindh** [vijendra1125/Custom-Mask-RCNN-using-Tensorfow-Object-detection-API#3]
Input [CropAndResize/crop_size] for node [CropAndResize] not found in function 'getConstBlob'
this error arise...?

**Comment by vijendra1125** [comment]
Information is not enough, give complete error log

**Comment by ganesharavindh** [comment]
when I try to load model in opencv, this error comes

          net = cv.dnn.readNetFromTensorflow(modelWeights, textGraph);
cv2.error: OpenCV(3.4.4) /io/opencv/modules/dnn/src/tensorflow/tf_importer.cpp:535: error: (-2:Unspecified error) Input [CropAndResize/crop_size] for node [CropAndResize] not found in function 'getConstBlob'

**Comment by vijendra1125** [comment]
Let me know if you are following the instruction given as it is or if you have tried something different. If you are following the instructions as it is then mostly it is problem with the version of opencv you are using.

In case you are trying something different then give me enough detail so that i could reproduce this error on my system.

**Comment by ganesharavindh** [comment]
I followed the instruction given by yours.The model works very well for tensorflow libraries. For opencv a .pbtxt file is needed.

Pretrained mask_rcnn model is trained for 90 classes so they created .pbtxt file for 90 classes, for custom trained model I already tried [this](https://github.com/opencv/opencv/wiki/TensorFlow-Object-Detection-API). For ssd_mobilenet_v1_coco model the generated .pbtxt file suites for opencv by just replacing the classes in .pbtxt file.I don't know how to generate .pbtxt file for custom trained mask_rcnn model, suggest me a few ideas to work with opencv.

opencv version : 3.4.4
numpy version : 1.16.0

**Comment by vijendra1125** [comment]
I have not tried to work with opencv libraries for object detection so cant give you idea regarding opencv. But i saw simillar issue being discussed on opencv GitHub repository, you might find solution there or try opening a new issue there.
